.github/workflows/pr-test.yml

```
@@ -233,7 +241,10 @@ jobs:
           BUILD_NUMBER: ${{ github.run_number }}
 
       - name: Run Test
-        run: bash ci/bash.sh ${DOCKER_IMAGE} --no-gpu ./scripts/task_test_jit_cache_package_build_import.sh
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:17: Double quote to prevent globbing and word splitting

(shellcheck)